added ground-markings (#48)
iXtenda/iVP-gitbook: home/ivp-planning/advanced-tools/ground-markings.md

# Ground Markings
 
It is possible to convert your path into a ground marking, enabling you to highlight specific areas with color-coding, which will be visible in the [2D-panel](../user-interface/the-2d-panel.md) and [3D-panel](../user-interface/the-3d-panel.md). To do so, simply change the [paths](path-tool.md) type to **Ground Marking** in the [properties panel](../user-interface/the-info-panel.md).
 
![](../../../.gitbook/assets/path_tool_ground_marking_type.jpg)

Besides the regular path options, you will also have the option to change the ground markings **line width**, **alignment**, **gap** and its **color** in the [properties panel](..user-interface/..the-info-panel.md)  

* **Line width**: Determines the width of the ground markings outer and inner lines.

> * **Alignment**: Adjusts the alignment of the ground markings inner and outer lines to *Center*, *Inside* or *Outside*.

Reviewer: This could be explained in a bit more detail.

> * **Gap**: Adjusts the size of the gap between the inner and outer lines.

Reviewer: ```suggestion
* **Gap**: Adjusts the distance between the inner and outer lines.
```

# Ground Markings
 
It is possible to convert your path into a ground marking, enabling you to highlight specific areas with color-coding, which will be visible in the [2D-panel](../user-interface/the-2d-panel.md) and [3D-panel](../user-interface/the-3d-panel.md). To do so, simply change the [paths](path-tool.md) type to **Ground Marking** in the [properties panel](../user-interface/the-info-panel.md).
 
![](../../../.gitbook/assets/path_tool_ground_marking_type.jpg)

Besides the regular path options, you will also have the option to change the ground markings **line width**, **alignment**, **gap** and its **color** in the [properties panel](..user-interface/..the-info-panel.md)  

* **Line width**: Determines the width of the ground markings outer and inner lines.

* **Alignment**: Adjusts the alignment of the ground markings inner and outer lines to:  

> Center: The marking is aligned parallel to the drawn path through its center. The yellow lines are positioned parallel to each other, with the drawn path situated between them.  

Reviewer: ```suggestion
Center: The marking is aligned to the drawn path with its center. The yellow lines are positioned parallel to each other, with the drawn path situated between them.  
```

Inside: The marking is inside the drawn path. The outer line is identical to the drawn line.  

> Outside: The marking is outside the  drawn path. The inside line is identical to the drawn line.

Reviewer: ```suggestion
Outside: The marking is outside the drawn path. The inner line is identical to the drawn line.
```
